examples/create_or_get_Stream.py

```diff
@@ -0,0 +1,23 @@
+from streamr.client.client import Client
+
+# To create or get a stream you should create a client at first
+myClient = Client({'apiKey': 'your-api-key'})
+
+# To create a new stream, you can use 'createStream' or 'getOrCreateStream' with a 'name' parameter
+# Note that createStream is running forcefully.
+# That means you can create two stream with same name but the streamIds are different
+
+
```

Comment:
add stream_id = 'stream-test'

Reply:
We are not sure what do you mean. We think this issue is the same as the previous one. The streamId should be a 22 bytes string, which can be found in the stream page, also can be obtained using the 'getStreamByName' method.